overlord/snapstate/snapstate.go

```diff
@@ -452,14 +452,30 @@ func defaultContentPlugProviders(st *state.State, info *snap.Info) []string {
 	return out
 }
 
+func getBoolFeatureFlag(tr *config.Transaction, flag string) (bool, error) {
```

Comment:
if I look at how we call other functions like this, it would be getFeatureFlagBool 
the other approach is to assume flag means bool (though that is not how go stdlib does it) and just call it getFeatureFlag